website/react-magma-docs/src/pages/data-visualization/chart-demos.mdx

```diff
@@ -0,0 +1,1829 @@
+      {
+        product: 'Product 1',
+        feature: 'Quality',
+        score: 60,
+      },
+      {
+        product: 'Product 2',
+        feature: 'Price',
+        score: 70,
+      },
+      {
+        product: 'Product 2',
+        feature: 'Usability',
+        score: 63,
+      },
+      {
+        product: 'Product 2',
+        feature: 'Availability',
+        score: 78,
+      },
+      {
+        product: 'Product 2',
+        feature: 'Performance',
+        score: 50,
+      },
+      {
+        product: 'Product 2',
+        feature: 'Quality',
+        score: 30,
+      },
+    ],
+    options: {
+      title: 'Radar',
+      radar: {
+        axes: {
+          angle: 'feature',
+          value: 'score',
+        },
+      },
+      data: {
+        groupMapsTo: 'product',
+      },
+      height: '400px',
+    },
+  };
+
+  return (
+    <Card style={{ padding: '12px' }}>
+      <CarbonChart {...args} />
+    </Card>
+  );
+}
+```
+
+<MoreDemoesLink
+  number="2"
+  path="https://storybook-preview-1544--upbeat-sinoussi-f675aa.netlify.app/?path=/story/carbonchart-radar--radar"
+/>
+
+## Scatter
+
+Scatter plots use data points to plot two measures along a scale, allowing exploration of correlations between different variables.
+
+```tsx
+import React from 'react';
+import { Card, CarbonChartType } from 'react-magma-dom';
+
+function Example() {
+  const args = {
+    type: CarbonChartType.scatter,
+    dataSet: [
+      {
+        group: 'Dataset 1',
+        employees: 5000,
+        sales: 32100,
+      },
+      {
+        group: 'Dataset 1',
+        employees: 3000,
+        sales: 25100,
+      },
+      {
+        group: 'Dataset 1',
+        employees: 8000,
+        sales: 12100,
+      },
+      {
+        group: 'Dataset 1',
+        employees: 4000,
+        sales: 53100,
+      },
+      {
+        group: 'Dataset 2',
+        employees: 5000,
+        sales: 32100,
+      },
+      {
+        group: 'Dataset 2',
+        employees: 2000,
+        sales: 34100,
+      },
+      {
+        group: 'Dataset 2',
+        employees: 4000,
+        sales: 23100,
+      },
+      {
+        group: 'Dataset 2',
+        employees: 7000,
+        sales: 14100,
+      },
+      {
+        group: 'Dataset 2',
+        employees: 6000,
+        sales: 53100,
+      },
+    ],
+    options: {
+      title: 'Scatter (linear x & y)',
+      axes: {
+        bottom: {
+          title: 'No. of employees',
+          mapsTo: 'employees',
+          scaleType: 'linear',
+        },
+        left: {
+          title: 'Annual sales',
+          mapsTo: 'sales',
+          scaleType: 'linear',
+        },
+      },
+      height: '400px',
+    },
+  };
+
+  return (
+    <Card style={{ padding: '12px' }}>
+      <CarbonChart {...args} />
+    </Card>
+  );
+}
+```
+
+<MoreDemoesLink
+  number="5"
+  path="https://storybook-preview-1544--upbeat-sinoussi-f675aa.netlify.app/?path=/story/carbonchart-scatter--scatter-linear-x-and-y"
```

Comment:
I know it would be a broken link until it gets merged, but should we just update `1544` to dev so we don't have to do it as a follow up? We know the URL will work as soon as this is merged